README.rst

```diff
@@ -1,13 +1,13 @@
 .. -*- mode: rst -*-
 
-|Travis|_ |AppVeyor|_ |Codecov|_ |rtd|_  |pypi_version|_  |python_version|_ |gitter|_ |saythanks|_
+|Azure|_ |Travis|_ |Codecov|_ |rtd|_ |pypi_version|_ |python_version|_ |gitter|_
+
+.. |Azure| image::https://dev.azure.com/franciscode-la-pena-manchon/hyperspy/_apis/build/status/hyperspy.hyperspy?branchName=RELEASE_next_minor
```

Comment:
Missing space between ``::`` and ``https``?

Reply:
Thanks @francisco-dlp  - fixed